shell/deepdive-run

```diff
@@ -108,6 +108,9 @@ fullConfig=$run_dir/deepdive.conf
         echo 1
     ) 2>/dev/null)}
     export PARALLELISM
+    
+    # set INPUT_BATCH_SIZE env var 
+    ! [[ -e "$INPUT_BATCH_SIZE" ]] || export INPUT_BATCH_SIZE=10000
```

Comment:
No worries.  Can you also update the ddlog submodule to point to the commit relevant to this?